lib/http.js

```diff
@@ -335,10 +337,24 @@ async function _getMeterClient() {
 async function _createMeter({controller, productId}) {
   const {url} = config['profile-http'].meterService;
 
+  // create root capability
+  const capability = `urn:zcap:root:${encodeURIComponent(url)}`;
+
+  // create signer using the application's capability invocation key
+  const {keys: {capabilityInvocationKey}} = getAppIdentity();
+  const invocationSigner = capabilityInvocationKey.signer();
```

Comment:
Did you consider using ezcap instead? I think that's the preferred method now.

Reply:
Sure: https://github.com/digitalbazaar/bedrock-profile-http/pull/39/commits/dbd4b6377db7470404d0ead9acd1cdc32bbde45f